_posts/blog/2017-08-09-interconnection_update.md

```diff
@@ -0,0 +1,76 @@
+
+M-Lab’s focus on interconnection was initially inspired by our observation of relative underperformance in cities where the platform maintained multiple measurement sites. Using a straightforward research method and open data, M-Lab compared trends across the US to identify patterns of performance degradation that occurred between prominent access and transit ISPs. We defined “degradation” as a drop in download throughput, an increase in round trip time, or an increase in packet retransmission rate, relative to past performance of that network and other networks in the same location. To avoid spurious reports, the incidents described were those where substantial degradation occurred, rather than minor disruptions. These episodes had clear fingerprints: observed degradation was nearly always diurnal, with significantly worse performance during peak use hours (defined by the FCC as between 7pm and 11pm local time). Degraded performance co-occurring with peak usage suggested that network load was causing congestion that lead to poor performance. These episodes of poor performance frequently continued every day for months without being fixed, which further pointed towards systemic network under-provisioning as a cause.
+
+In the Interconnection Study, we found a consistent theme across multiple access ISPs and transit ISPs: the interconnection relationships between network operators have a significant impact on broadband performance. M-Lab observed sustained degradation experienced by customers of the most prominent American broadband access ISPs: AT&T, Comcast, CenturyLink, Time Warner Cable, and Verizon. The degradation documented in the publication occurred when traffic passed over interconnections with transit ISPs Cogent Communications (Cogent), Level 3 Communications (Level 3), and XO Communications (XO) – networks where M-Lab had hosted measurement endpoints. We observed similar patterns of performance degradation whenever and wherever specific pairs of access and transit ISPs interconnected. While the onset of congestion and end consumer impact differed between each access and transit provider relationship, the sustained episodes documented in the report typically began in early 2013, and continued into the following year.
+
+![Verizon across Internap and Cogent in New York, January 2013 - January 2014, Download Throughput]({{ site.baseurl }}/images/blog/interconnection_update-Verizon-Internap_Cogent-NY.png){: width="838" height="260"}
```

Comment:
Linking on the image works if the image is not inside a table. If you want them all consistently in a table with the caption in a bottom row, I can try just writing the whole table in html versus a mix of html and markdown.